### DISCUSSION TITLE
Capme equivalent

### DISCUSSION THREAD

**Discussion by RichHouston** [Security-Onion-Solutions/securityonion#2652] [General] [answered]
Sorry if this has been asked and answered. Hope I am posting in the right place.

We are looking to upgrade to SO 2.3, which looks awesome by the way, but we are addicted to Capme... Is there an equivalent in 2.3?

Thanks in advance and appreciate all the hard work that goes into this project.

Thanks

Rich

**Comment by TOoSmOotH** [comment, marked as answer]
Yes you can see the following screenshots:

https://1.bp.blogspot.com/-I2HsgcH_VSU/X-D0ktLAP0I/AAAAAAAAG20/v9QHzouzNbYQ5pwdXOlgT5UVDjHdNEd_QCLcBGAsYHQ/w640-h288/Screen%2BShot%2B2020-12-21%2Bat%2B11.59.27%2BAM.png

https://1.bp.blogspot.com/-LEiNYTdeCgM/X-H7EmbJPQI/AAAAAAAAG4U/MrqHcbvlWtkeI-SgwQqJV7afx1tv7EuAgCLcBGAsYHQ/w640-h236/Screen%2BShot%2B2020-12-22%2Bat%2B8.56.13%2BAM.png
